nucleus/annotation.py

```diff
@@ -70,6 +71,9 @@ def to_json(self) -> str:
         """Serializes annotation object to schematized JSON string."""
         return json.dumps(self.to_payload(), allow_nan=False)
 
+    def has_local_files(self) -> bool:
```

Comment:
This implementation is correct for all classes currently. It seems cleaner to me to assume no local files by default, since getting that working is always going to require a bunch of other code.

Added some docstring.